storybook/storybook-css/src/components/Heading/Heading.stories.tsx

```diff
@@ -0,0 +1,78 @@
+/* @license CC0-1.0 */
+
+import type { Meta, StoryObj } from '@storybook/react'
+import { Heading } from './Heading'
+
+const meta = {
+  title: 'CSS Component/Heading',
+  id: 'css-heading-1',
+  component: Heading,
+  argTypes: {
+    textContent: {
+      table: {
+        type: { summary: 'string' },
+      },
+    },
+    level: {
+      table: {
+        disable: true,
+      },
+    },
+    styleLevel: {
+      options: [1, 2, 3, 4, 5, 6],
+      control: 'select',
+      table: {
+        type: { summary: 'number' },
+        defaultValue: { summary: '1' },
+      },
+    },
+  },
+  args: {
+    textContent: 'Jouw typograaf biedt mij zulke exquise schreven!',
+    level: 1,
+    styleLevel: 1,
+  },
+  tags: ['autodocs'],
+  parameters: {
+    tokensPrefix: 'amsterdam-heading',
+    docs: {
```

Comment:
Docs doorgenomen en je kunt hier nu `code` en `transform` meegeven, ga ik mee aan de slag.

Reply:
We have a story for this in Jira.